vm/actor/tests/paych_actor_test.rs

```diff
@@ -0,0 +1,1115 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod common;
+use actor::{
+    paych::{
+        ConstructorParams, LaneState, Merge, Method, ModVerifyParams, PaymentVerifyParams,
+        SignedVoucher, State as PState, UpdateChannelStateParams, LANE_LIMIT, SETTLE_DELAY,
+    },
+    ACCOUNT_ACTOR_CODE_ID, INIT_ACTOR_ADDR, INIT_ACTOR_CODE_ID, MULTISIG_ACTOR_CODE_ID,
+    PAYCH_ACTOR_CODE_ID, SYSTEM_ACTOR_ADDR,
+};
+use address::Address;
+use cid::Cid;
+use clock::ChainEpoch;
+use common::*;
+use crypto::Signature;
+use db::MemoryDB;
+use derive_builder::Builder;
+use encoding::to_vec;
+use ipld_blockstore::BlockStore;
+use message::UnsignedMessage;
+use num_bigint::BigInt;
+use vm::{ExitCode, Serialized, TokenAmount, METHOD_CONSTRUCTOR, METHOD_SEND};
+
+const R_PAYEE_ADDR: u64 = 103;
+const R_PAYER_ADDR: u64 = 102;
+struct LaneParams {
+    epoch_num: ChainEpoch,
+    from: Address,
+    to: Address,
+    amt: i64,
+    lane: u64,
+    nonce: u64,
+}
+
+fn is_ok<'a, BS: BlockStore>(rt: &mut MockRuntime<'a, BS>, method_num: u64, ser: &Serialized) {
+    assert!(rt.call(&*PAYCH_ACTOR_CODE_ID, method_num, ser).is_ok());
+}
+
+fn expect_error<'a, BS: BlockStore>(
+    rt: &mut MockRuntime<'a, BS>,
+    method_num: u64,
+    ser: &Serialized,
+    exp: ExitCode,
+) {
+    assert_eq!(
+        exp,
+        rt.call(&*PAYCH_ACTOR_CODE_ID, method_num, ser,)
+            .unwrap_err()
+            .exit_code()
+    );
+}
+
+mod paych_constructor {
```

Comment:
why do you put tests in seperate modules? doesn't do anything afaik?

Reply:
Well its only really to mimic the format they have in GO where they can run multiple tests in a func. Other than that by keeping them in diff modules helps put funcs/constants that are used in certain tests together